hoth_api.yaml

```diff
@@ -187,6 +187,38 @@ components:
         maxAllowed:
           type: number
           description: Represents the maximum allowed size to return
+    ProvisioningClassificationLevel:
+      description: >-
+        Represents the classification level for this environment. Attempts to change this field via PUT or PATCH
+        should result in a HTTP 400.
+      type: string
+      enum:
+        - UNCLASSIFIED
+        - SECRET
+        - TOP_SECRET
```

Comment:
In SNOW and in the HOTH API I think the general `enum` used for classification is `U`, `S`, and `TS`. Should we start shifting to using the full name?

Reply:
Let's leave it as-is for now and make a single PR to fix this if that's where we want to go.

Reply:
For this API I like the full names